docs/content/en/docs/reference/changelog.md

```diff
@@ -2,13 +2,43 @@
 linkTitle: "Changelog"
```

Comment:
Sorry, I think I wasn't clear before. You can remove lines 2, 3, and 4, I think.